File: Weekend-Farm-User/app/src/main/java/com/farm/weekend/gram/weekend_farm_user/util/UtilClass.java
package com.farm.weekend.gram.weekend_farm_user.util;

import android.content.Context;
import android.widget.Toast;

public final class UtilClass {

    public static final String TOKEN = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI1MWI3NTQ4OC1jZTg0LTRiNzEtYjgzYi0zNzgwMDVkODg0NDQiLCJmcmVzaCI6ZmFsc2UsInR5cGUiOiJhY2Nlc3MiLCJpZGVudGl0eSI6ImFiYyIsImlhdCI6MTUzNTczNzI5MiwibmJmIjoxNTM1NzM3MjkyLCJleHAiOjE1MzgzMjkyOTJ9.7bG0O8YLxYTMdt8WyTR76LV_4d7wB8kFvt4hHpBKrFc";

    public static void Toast(Context context, String message){
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
    }
}

File: Weekend-Farm-Admin/app/src/main/java/com/farm/weekend/gram/weekend_farm_admin/util/UtilClass.kt
package com.farm.weekend.gram.weekend_farm_admin.util

import android.content.Context
import android.content.SharedPreferences
import android.widget.Toast

object UtilClass {
    fun showToast(context: Context, message: String) = Toast.makeText(context, message, Toast.LENGTH_SHORT).show()

    private fun getPref(context: Context): SharedPreferences {
        val preference = context.getSharedPreferences("PREFERENCE", Context.MODE_PRIVATE)
        return preference
    }

    fun saveToken(context: Context, token: String){
        val editor = getPref(context).edit()
        editor.putString("Authorization", token)
        editor.apply()
    }

    fun removeToken(context: Context){
        val editor = getPref(context).edit()
        editor.remove("Authorization")
        editor.apply()
    }

    fun getToken(context: Context): String{
        return "JWT " + getPref(context).getString("Authorization", "")
    }

    fun delayHandler(runnable: Runnable, delayTime: Long) {
        var handler = android.os.Handler()
        handler.postDelayed(runnable, delayTime)
    }
}